client/src/main/java/io/pravega/client/stream/RetentionPolicy.java

```diff
@@ -95,4 +100,13 @@ public static RetentionPolicy bySizeBytes(long sizeMin, long sizeMax) {
         return RetentionPolicy.builder().retentionType(RetentionType.SIZE)
                 .retentionParam(sizeMin).retentionMax(sizeMax).build();
     }
+
+    /**
+     * Create a retention policy to configure a stream to be used for internal streams.
```

Comment:
Please document what it does. Not when it is used. The former needs to be explained the later can be infered from code.